app/src/main/java/my/noveldokusha/scraper/sources/ReadWn.kt

```diff
@@ -37,12 +37,11 @@ class ReadWn : SourceInterface.catalog
 
     override suspend fun getChapterList(doc: Document): List<ChapterMetadata>
```

Comment:
I get 403 when trying to get catalog or search, the chapters also don't have title. Must be fixed.

Reply:
Don't know how to fix it.